tests/appsec/contrib_appsec/utils.py

```diff
@@ -802,11 +811,30 @@ def test_request_suspicious_request_block_match_request_body(
         ],
     )
     @pytest.mark.parametrize(
-        "headers",
-        [{"Accept": "text/html"}, {"Accept": "text/json"}, {}],
+        ["headers", "use_html"],
+        [
+            ({"Accept": "text/html"}, True),
+            ({"Accept": "application/json"}, False),
+            ({}, False),
+            ({"Accept": "text/*"}, True),
+            ({"Accept": "text/*;q=0.8, application/*;q=0.7, */*;q=0.9"}, True),
+            ({"Accept": "text/*;q=0.7, application/*;q=0.8, */*;q=0.9"}, False),
+            ({"Accept": "text/html;q=0.9, text/*;q=0.8, application/json;q=0.85, */*;q=0.9"}, True),
+        ],
     )
     def test_request_suspicious_request_block_custom_actions(
-        self, interface: Interface, get_tag, asm_enabled, metastruct, root_span, query, status, rule_id, action, headers
+        self,
+        interface: Interface,
+        get_tag,
+        asm_enabled,
+        metastruct,
+        root_span,
+        query,
+        status,
+        rule_id,
+        action,
+        headers,
+        use_html,
```

Comment:
nit: Should we be passing this in hardcoded? Looks like we could also validate the behavior of `_use_html()` and/or `_ctype_from_headers()` here, but if we already have enough coverage for that then feel free to ignore this.

Reply:
we are covering the usage of `_ctype_from_headers` here with the new `Accept` headers value that could only lead to the expected answer with the appropriate computation from `_ctype_from_headers`
There will also be tested on system tests